package.json

```diff
@@ -21,7 +21,16 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "prepare": "husky",
+    "lint": "eslint .",
```

Comment:
this amount of scripts is redundant, usually we using lint:check, lint:fix, format:check and format:fix scripts.